### PR TITLE
Update fr.po

### PR DESCRIPTION
**Pull request by Jean-Gabriel1982**

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -2964,26 +2964,26 @@ msgstr "Paramètres du programme"
...
 
-msgid "Defaults and Initial Settings"
-msgstr "Valeurs initiales et paramètres par défaut"
+msgid "Paths"
+msgstr "Chemins"
+
+msgid "Audio"
+msgstr "Audio"
 
 msgid "Reverb"
```
Comment (oleg68) [inline po/fr.po:2973]: I suggest you to change the order of the msgid for reflecting the tab order in the dialog.

File: po/fr.po [hunk 2]
```diff
@@ -2964,26 +2964,26 @@ msgstr "Paramètres du programme"
...
 msgid "MIDI Devices"
 msgstr "Interfaces MIDI"
 
 msgid "Temperaments"
 msgstr "Tempéraments"
 
-msgid "Initial MIDI Configuration"
-msgstr "Configuration MIDI initiale"
+msgid "Initial MIDI"
+msgstr "Config.MIDI initiale"
```
Comment (oleg68) [inline po/fr.po:2983]: Very long. For avoiding OSx issues with wrapping.

```suggestion
msgstr "MIDI initiale"
```

**Review by oleg68 — APPROVED** [review]

**Review by oleg68 — CHANGES_REQUESTED** [review]
@Jean-Gabriel1982 please add an entry to the CHANGELOG.md describing your changes

**Comment by larspalo** [comment]
@oleg68 Please, updating translation is not introducing anything noteworthy so there's really no need to add a CHANGELOG entry for that. It's another thing if a new language is added.

**Comment by rousseldenis** [comment]
> @oleg68 Please, updating translation is not introducing anything noteworthy so there's really no need to add a CHANGELOG entry for that. It's another thing if a new language is added.

I agree.

Thinking about it, we could imagine an external translation service such as weblate. That allows external contributors to translate the software. https://weblate.org/fr/

**Review by rousseldenis — APPROVED** [review]

**Comment by oleg68** [comment]
@Jean-Gabriel1982 Based on your PR I've made a new one: #986 whith adding in French all untranslated items in the settings dialog

**Pull request closed without merging**